renderer/index.js

```diff
@@ -153,6 +153,11 @@ function cleanUpConfig () {
       delete ts.posterURL
       ts.posterFileName = infoHash + extension
     }
+
+    // Migration: add per-file selections
+    if (!ts.selections) {
+      ts.selections = ts.files.map((x) => true)
```

Comment:
Maybe in a future version. Let's not mess with the config right before this release.